### ISSUE TITLE
Install command is missing.

### ISSUE THREAD

**Issue by fubarhouse** [yyuu/pyenv#777]
I'm unsure as to why, but does anybody know why the install/uninstall commands are simply missing?

````
pyenv 1.0.4-11-ga73d90b
Usage: pyenv <command> [<args>]

Some useful pyenv commands are:
   commands    List all available pyenv commands
   local       Set or show the local application-specific Python version
   global      Set or show the global Python version
   shell       Set or show the shell-specific Python version
   rehash      Rehash pyenv shims (run this after installing executables)
   version     Show the current Python version and its origin
   versions    List all Python versions available to pyenv
   which       Display the full path to an executable
   whence      List all Python versions that contain the given executable

See `pyenv help <command>' for information on a specific command.
For full documentation, see: https://github.com/yyuu/pyenv#readme
````

**Comment by yyuu** [comment]
install is a command from python-build plugin. It may not be available if your pyenv installation is not configured properly into your shell. Typically you need to add 'eval "$(pyenv init -)"' in your shell's initialization script.

**Comment by fubarhouse** [comment]
Thanks, that at least gives me a starting point.

**Comment by the-moog** [comment]
To help others, I had this issue and found that in the .bashrc rather than as documented I need to add 'export' to the PYENV_ROOT and PATH variable creation.
e.g.

```
export PYENV_ROOT="$HOME/.pyenv"
export PATH="$PYENV_ROOT/bin:$PATH"

if command -v pyenv 1 > /dev/null 2>&1; then
  eval "$(pyenv init -)"
fi
```
I think this is something to do with the way the login shell is being created.

**Comment by tayjaybabee** [comment]
### To help users in the future who might have this problem, regardless of shell:

### I was receiving:
```  
taylor@taylor-Precision-M6600  ~  pyenv install 3.5.3                                                                 ✔  8229  09:05:09  
pyenv: no such command `install'
```

  ...after adding the required lines to my .zshrc file one-at-a-time (copying them from some tutorial or w/e), and then re-initialized my shell

### What caused this?
  Turns out I had not copy-pasted the entirety of the required line each time, leaving off the final `"` of each. After leaving this file malformed in this way I was still able to run `source ~/.zshrc` with no complaints from the shell. This lack of complaint from the shell upon re-initialization left me dumbfounded for a solution to this for some time. 

### TL;DR, by Chance?
  The error `pyenv: no such command 'install'` can be caused (or was in my case, at least) by malformed Path export commands in your shell's configuration. Checking this is minimally invasive. If you copy the [except posted by](https://github.com/pyenv/pyenv/issues/777#issuecomment-682723484) @the-moog (above) and use that to replace the malformed lines you might find your solution.
